### ISSUE TITLE
Blurry text for overlays and client plugin list in .jar, but not .exe

### ISSUE THREAD

**Issue by chaojimbo** [runelite/runelite#11826]
**Overview**
Starting with update 1.6.14 on May 9th, various text in the game and client have become blurry for me. It was very noticeable with the ammunition and Miscellania overlays and I can see that there is some sort of scaling occurring for other text. I cannot fix the text despite turning off all of the plugins, so **it seems that the blurry text is not plugin related**. Under the RuneLite plugin, I have all text set to "small" which makes much of the text look proper. If they are set to "regular", then the text is blurry in those cases. 

I play the game in stretched fixed mode and with the GPU plugin turned on. Disabling these plugins does not fix the text, even after restarting the client. The text in the plugin list also has slight blur to them. 

When researching the issue, people have said it has to do with Windows' scaling settings when you have multiple displays and one is set above 100% scale. Both of my displays are 1080p monitors that run at 144 Hz and Windows has default DPI and 100% scale set to both. 

I also run the game from the .jar file instead of the .exe. 
**I just realized that the 64-bit .exe version of RuneLite doesn't have this issue. I have been using the ,jar for years and right when I was about to post this Github post, I decided to include this tidbit about using the .jar, which made me curious to try the .exe version.**

See screenshots for comparisons of blurry and normal text.

**Environment**
 - OS: Windows 10 Pro 1909 x64
 - RuneLite version: 1.6.18.1
 - Launcher version: 1.6.1
 - Java version: Version 8 Update 251 (I was originally on update 197-- same issue.)
 - Two monitors: 1920x1080, 100% scale. 
 - NVIDIA GTX 1070 FTW with driver version 446.14

**Screenshots**

![image](https://user-images.githubusercontent.com/10885788/83927647-673db980-a742-11ea-99e4-f7ef858f752f.png)
Any text shown in a box like this (ammunition, timers, etc) will have blurry numbers.

![image](https://user-images.githubusercontent.com/10885788/83927678-850b1e80-a742-11ea-9af0-e37e3052e274.png)
This one is harder to see, but the plugin list is blurry.

![image](https://user-images.githubusercontent.com/10885788/83928268-fbf4e700-a743-11ea-9fd1-79f669140763.png)
The top one is the .jar, the bottom one is the 64-bit .exe. 

![image](https://user-images.githubusercontent.com/10885788/83927712-96ecc180-a742-11ea-8d80-269b1dbb1251.png)
Hover-over tooltip text is clear when the text size is set to "small"

![image](https://user-images.githubusercontent.com/10885788/83927748-a9ff9180-a742-11ea-9f28-bc29b0b79466.png)
Here is the same thing but set to "Regular"

![image](https://user-images.githubusercontent.com/10885788/83927767-b71c8080-a742-11ea-9c8d-e8ea9a06a063.png)
Small FPS, non-blurry. 

![image](https://user-images.githubusercontent.com/10885788/83927778-c3a0d900-a742-11ea-9714-69aa10c00e96.png)
Regular FPS, blurry.

![image](https://user-images.githubusercontent.com/10885788/83928040-36aa4f80-a743-11ea-82b5-ce56c0ae513d.png)
World map has some blurry tooltips for certain things...

![image](https://user-images.githubusercontent.com/10885788/83928059-46299880-a743-11ea-81c3-79121f2f213e.png)
But not everything...

**Comment by Nightfirecat** [comment]
Can you provide copies of your `launcher.log` and `client.log` files when running the jdk versus the exe?

**Comment by chaojimbo** [comment]
> 
> 
> Can you provide copies of your `launcher.log` and `client.log` files when running the jdk versus the exe?

Sure. 

[launcher.log](https://github.com/runelite/runelite/files/4738906/launcher.log)
[client.log](https://github.com/runelite/runelite/files/4738907/client.log)
This was after running the jar.

[launcher.log](https://github.com/runelite/runelite/files/4738912/launcher.log)
[client.log](https://github.com/runelite/runelite/files/4738913/client.log)
Here is the exe.

**Comment by chaojimbo** [comment]
Hmm, I installed the latest JDK which fixed the blur issue. Sorry for wasting time, I guess my JDK was just really outdated (possibly ~4 years old) and it wasn't playing nicely with RuneLite. Didn't realize JDK was different than standard Java. :\ 

Should I close the thread?

**Comment by Hydrox6** [comment]
To be honest, if the Exe works, you should really be using it, since the version of Java it comes with is something that we target and that works for the vast majority of users.

**Comment by chaojimbo** [comment]
> 
> 
> To be honest, if the Exe works, you should really be using it, since the version of Java it comes with is something that we target and that works for the vast majority of users.

I believe the .exe didn't work for me when I first made the switch to RuneLite years ago which is why I've been using the same .jar ever since. The .exe is working now though (and I assume it was just Windows 10 being a butt when I first tried back then). I'll keep that in mind though-- RuneLite generally has no issues for me otherwise. Thank you. :)
